runtimes/ruby-3.3/src/server.rb

```diff
@@ -310,6 +319,10 @@ def handle(request, response)
   handle(request, response)
 end
 
+trace '*' do
+  handle(request, response)
+end
+
```

Comment:
what about head requests for ruby? I dont see a handler for it?

Reply:
The `get` handler auto-registers one for `head` too, [here's the reference](https://github.com/sinatra/sinatra/blob/main/lib/sinatra/base.rb#L1524C7-L1532C10).

Its been a while since I tested ruby, but the `head` handler's request are probably overriden with the `get`.